src/orange/SenseUtils.hh

```diff
@@ -120,13 +120,16 @@ inline char const* to_cstring(SignedSense s)
  */
 class SenseValue
 {
-  private:
-    enum : char
+    enum class Bits : char
     {
-        sense_bit,
-        is_assigned_bit,
+        sense,
+        is_assigned,
+        size_
     };
 
+    static inline constexpr auto sense_bit = Bits::sense;
```

Comment:
is inline needed?

Reply:
Apparently not... from cppreference:
> A constexpr specifier used in the first declaration of a function or static data member(since C++17) implies inline

Guess I was just too used to using C++14!